charts/plex-media-server/templates/statefulset.yaml

```
@@ -30,6 +30,11 @@ spec:
       priorityClassName: {{ .Values.priorityClassName | quote }}
       {{- end }}
       volumes:
+      {{- if .Values.pms.configExistingClaim }}
+      - name: pms-config
+        persistentVolumeClaim:
+          claimName: {{ tpl .Values.pms.configExistingClaim . }}
```

Do we need to template the string here, or can we use the value directly instead?

```suggestion
          claimName: {{ .Values.pms.configExistingClaim | quote }}
```